spring-integration-core/src/test/java/org/springframework/integration/configuration/EnableIntegrationTests.java

```diff
@@ -135,6 +137,8 @@
 @DirtiesContext
 public class EnableIntegrationTests {
 
+	private static ThreadLocal<Boolean> lifecycleStopped = new ThreadLocal<>();
```

Comment:
Polish looks good, thanks; definitely better. Let's leave the docs as-is (conservative).